src/Test.FunctionalTests.Sarif/Multitool/ValidateCommandTests.cs

```diff
@@ -465,15 +465,20 @@ private string MakeTestFileName(ReportingDescriptor rule, string testFileNameSuf
 
         protected override string ConstructTestOutputFromInputResource(string inputResourceName, object parameter)
         {
-            string v2LogText = GetResourceText(inputResourceName);
+            string v2LogText = GetInputSarifTextFromResource(inputResourceName);
 
-            string inputLogDirectory = this.OutputFolderPath;
+            string inputLogDirectory = this.TestOutputDirectory;
             string inputLogFileName = Path.GetFileName(inputResourceName);
-            string inputLogFilePath = Path.Combine(this.OutputFolderPath, inputLogFileName);
+            string inputLogFilePath = Path.Combine(this.TestOutputDirectory, inputLogFileName);
 
             string actualLogFilePath = Guid.NewGuid().ToString();
 
-            string ruleUnderTest = Path.GetFileNameWithoutExtension(inputLogFilePath).Split('.')[1];
+            // Splits a test file name such as 'GH1001.ProvideRequiredLocationProperties.cs'
```

Comment:
>Splits [](http://example.com/codeflow?start=15&length=6)

Please note that I've added a good comment to the code explaining this mysterious parse and retrieval of the 0th token. T